examples/jenkins-master/README.md

```diff
@@ -0,0 +1,92 @@
+Jenkins Master->Slave Example
+=============================
+This sample walks through the process of starting up an OpenShift cluster and deploying a Jenkins Pod in it.
+It also leverage the [kubernetes-plugin](https://wiki.jenkins-ci.org/display/JENKINS/Kubernetes+Plugin) to manage
+the Jenkins slaves that run as on-demand Pods.
+
+The sample job that this Jenkins example contains will execute an unit test for
+a simple Ruby application. The unit test will be executed in a pod managed by
+the kubernetes plugin.
+
+Because the Jenkins Slave pod needs an Docker image that connects to the Jenkins
+when it starts, this sample also provide a template that allows you to convert
+any OpenShift S2I image to a valid Jenkins Slave image.
+
+This template is not required, if you already have Docker image that has the
+script that launch the [slave
+agent](https://wiki.jenkins-ci.org/display/JENKINS/Distributed+builds#Distributedbuilds-Launchslaveagentheadlessly)
+as an entrypoint. In that case, you have to create the image stream for this
+image and use that image stream instead when creating the Jenkins master.
+
+Also note that the kubernetes plugin is already included in the official Jenkins
+image, so you don't have to use the template in this example. The template in
+this example will customize the Jenkins to install Git plugin and the sample
+Jenkins job used in this example.
+
+Steps
+------
+1. Create new OpenShift project, where the Jenkins server will run:
+```
+$ oc new-project ci
+```
+
+2. Give the Jenkins Pod [service account](https://docs.openshift.org/latest/admin_guide/service_accounts.html)
+   rights to do API calls to OpenShift.  This allows us to do the Jenkins Slave
+   image discovery automatically.
+```
+$ oc policy add-role-to-user edit system:serviceaccount:ci:default -n ci
+```
+
+3. Install the provided OpenShift templates:
+```
+# Slave convertor (optional):
+$ oc create -f https://goo.gl/YROngT
+
+# Jenkins master template:
+$ oc create -f https://goo.gl/HPXiOs
```

Comment:
the full link is way more ugly and it really sux when you want to copy&paste it from the README.md